### PR TITLE
feat: refdata API

### PR DESCRIPTION
**Pull request by fanzhao1986**
Add refdata wrapper to fetch refdata from crossfire API.

### CODE REVIEW

**Comment by fanzhao1986** [comment]
Hi @marayl , I found you add some text in README.md. Should do some refactor work. There are some instructions for developer not for user, those parts may move to `docs/dev/` directory.

**Comment by marayl** [comment]
> Hi @marayl , I found you add some text in README.md. Should do some refactor work. There are some instructions for developer not for user, those parts may move to `docs/dev/` directory.

Sounds good. Feel free to reorganise.

File: src/rex/client.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        url: url
+            crossfire trading platform api url.
+        """
+        key = key if key is not None else os.getenv("REACTIVE_API_TOKEN")
+        if key is None:
+            raise RuntimeError("no API key found")
+        self.__headers = {'Authorization': 'Bearer ' + key}
+        self.__url = url
+
+    def fetch_asset_reference(self):
```
Comment (marayl) [inline src/rex/client.py]: This function returns an instance of `AssetRefData`. Can we rename the method to `fetch_asset_ref_data`?

File: tests/util/test_cache.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#  Copyright (C) 2020 Reactive Markets Limited. All rights reserved.
```
Comment (marayl) [inline tests/util/test_cache.py]: Wrong license.

File: src/rex/domain/refdata.py [hunk 3]
```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
```
Comment (markmcdowell) [inline src/rex/domain/refdata.py]: Can you split this file up? It's too long.

File: docs/QuickStart.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
+.. _quickstart:
+
+==========
+Quickstart
+==========
+
+To use rex-py package to access crossfire trading platform, a API token must be ready. To see
```
Comment (markmcdowell) [inline docs/QuickStart.rst]: To use rex-py package to access exchange trading platform

**Comment by markmcdowell** [comment]
This is public, make sure to remove all private information and references to crossfire.

**Pull request merged**